packages/preact-iso/router.d.ts

```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent, VNode } from 'preact';
+
+export const LocationProvider: FunctionComponent;
+
+export function Router(props: { onLoadEnd?: () => void, onLoadStart?: () => void, children?: VNode[] }): VNode;
```

Comment:
I wonder if this should use ComponentChildren?

```suggestion
export function Router(props: { onLoadEnd?: () => void, onLoadStart?: () => void, children?: ComponentChildren }): VNode;
```

Reply:
I don't believe so? Otherwise something like a number would be a valid child. https://github.com/preactjs/preact/blob/1a0837fe887e2ee5d5b504d62d1738e71d2c76fc/src/index.d.ts#L46-L54

Reply:
That's actually allowed in Preact :D

Reply:
Seemed to break the router though... Might just need an optional chain in there.

Reply:
Ah! that is a very good point actually! VNode is best then.

Reply:
That's what I was thinking.